src/index.js

```diff
@@ -618,8 +628,8 @@ export default class Proctor {
       timestamp: `${new Date().toJSON().slice(0, 19).replace('T', ' ')} UTC`,
       disqualify: this.config[type].disqualify,
     };
-
-    if (this.config[type].showAlert) {
+    // to avoid 2 alerts, disabling alerts for browserBlur
+    if (this.config[type].showAlert && type !== VIOLATIONS.browserBlur) {
```

Comment:
Add a new config. Custom alert Message for browser blur and then use that here to show alert. 

Right now we are duplicating code + hard coding.